EasyPost/Models/API/CarrierFields.cs

```diff
@@ -14,7 +15,7 @@ public class CarrierFields : EasyPostObject
         ///     The credentials used in the production environment.
         /// </summary>
         [JsonProperty("credentials")]
-        public CarrierField? Credentials { get; set; }
+        public Dictionary<string, CredentialsField>? Credentials { get; set; }
```

Comment:
issue: I don't like the name change here (even if the new name may be more correct or readable). It moves this from a bug fix to a breaking change which we should avoid. We can fix the underlying deserialization without users needing to alter their code aside from a version bump.

Reply:
To be fair, because of this deserialization issue, a user has never been able to interact with a `CarrierField` object before, since every attempt to create one would have failed.

Reply:
Correct, though again, this breaking change will require end-users to make changes in their code if they've already deployed this - even if today it's not working. Let's not major version bump over something that should just be a bug fix patch release.